setup.cfg

```diff
@@ -107,6 +107,8 @@ asdf_schema_root = weldx/schemas
 #asdf_schema_skip_tests =
 #    weldx.bam.de/weldx/datamodels/single_pass_weld-1.0.0.schema.yaml
 norecursedirs = doc
+filterwarnings =
+    ignore::DeprecationWarning:traittypes.*:
```

Comment:
This syntax only filters all `DeprecationWarning`s from the `traittypes` module/namespace @vhirtham

Reply:
Good that you got it working. I tried this syntax too, but with the wrong module (`traitlets`)